assets/javascript/app.js

```diff
@@ -1,4 +1,43 @@
 //= require_directory ./vendor
 //
 
-// custom JS can go in this file, or another, then you add another require line
\ No newline at end of file
+// custom JS can go in this file, or another, then you add another require line
+
+function showCareerPrev(currIndex) {
+  'use strict';
+
+  var newIndex = parseInt(currIndex) - 1;
+  if (newIndex > 0) {
+    toggleCareer(currIndex, newIndex);
+  }
+}
+
+function showCareerNext(currIndex, count) {
```

Comment:
Just a nit, but "showNextCareer" reads a little more clearly to me than "showCareerNext". I know the latter makes the prefix standard, so this is one of those "better english" vs "more accurate" things. Not a blocker, but worth pointing out.